src/cli/call/procedure/Procedure.definition.ts

```diff
@@ -48,5 +48,9 @@ export const ProcedureDefinition: ICommandDefinition = {
             description: "Call a stored procedure and pass values for parameter indicators",
             options: "\"DEMO.SP2(?, ?)\" --parameters \"Hello\" \"world!\"",
         },
+        {
+            description: "Call a stored procedure and pass values for two output parameters. The first output requires a 2-character buffer. The second output is a message that will be truncated to the length of the placeholder.",
+            options: "\"DEMO.SP3(NULL, ?, ?)\" --parameters "00" "message_placeholder_message_placeholder"",
```

Comment:
All the quotes inside the string need to be escaped:
```javascript
options: "\"DEMO.SP3(NULL, ?, ?)\" --parameters \"00\" \"message_placeholder_message_placeholder\"",
```
Alternatively you can use backticks:
```javascript
options: `"DEMO.SP3(NULL, ?, ?)" --parameters "00" "message_placeholder_message_placeholder"`,
```

Either way should make the linter happy 🙂